pom.xml

```diff
@@ -339,8 +339,9 @@ See the Apache License Version 2.0 for the specific language governing permissio
         <plugin>
           <groupId>org.apache.felix</groupId>
           <artifactId>maven-bundle-plugin</artifactId>
-          <version>3.5.0</version>
+          <version>5.1.2</version>
           <configuration>
+            <excludeDependencies>jsr305</excludeDependencies>
```

Comment:
This removed the version range from the Package import `javax.annotation;version="[3.0,4)"`, which is great because one does not need the jsr305 in an OSGi runtime anymore.
Nevertheless there is still the Import-Package `javax.annotation` (without version range) remains.
If the `Nullable` annotation from `jsr305` is really not necessary at runtime I suggest to completely remove the import of the javax.annotation package, by adding a negated pattern to the `Import-Package` element.

If you are interested, I can provide a PR to change to that suggestion.